### PR TITLE
Sound manager

### PR DESCRIPTION
**Pull request by SassDude**
created audio manager prefab and interactions able to be done with the audio manager

### CODE REVIEW

**Review by GhostEntity12 — CHANGES_REQUESTED** [review]
Please ensure that the audio adheres to the [naming conventions](https://docs.google.com/document/d/1ngzQ6UUjyCyknk-3t2Rl_pwRQeF-trg98L_ouxy9t1c/edit#heading=h.1q0659lo19up) set out in the process document, and add some basic comments for readability and maintainability

File: Assets/Scripts/Managers/AudioManager.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+        foreach (Sound s in sounds)
+        {
+            s.source = gameObject.AddComponent<AudioSource>();
+            s.source.clip = s.clip;
+            
+            s.source.volume = s.volume;
+            s.source.pitch = s.pitch;
+            s.source.loop = s.loop;
+            s.source.playOnAwake = s.playOnAwake;
```
[the remark this answers is not in the document]
Comment (SassDude) [reply Assets/Scripts/Managers/AudioManager.cs:34]: good point, will do

File: Assets/Scripts/Managers/HuntingManager.cs [hunk 2]
```diff
@@ -29,6 +29,8 @@ protected override void Awake()
     private void Start()
     {
         huntingTimerText.text = FormatTime(huntingDurationSeconds);
+        audioManager = FindObjectsOfType<AudioManager>();
+        audioManager[0].Play("Ambience");
```
[the remark this answers is not in the document]
Comment (SassDude) [reply Assets/Scripts/Managers/HuntingManager.cs:33]: in the case we don't want music or sound effects to be cut off by each scene

File: Assets/Scripts/Managers/AudioManager.cs [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    {
+        if (instance == null) 
+            instance = this;
+        else 
+        {
+            Destroy(gameObject);
+            return;
+        }
+
+        DontDestroyOnLoad(gameObject);
```
[the remark this answers is not in the document]
Comment (SassDude) [reply Assets/Scripts/Managers/AudioManager.cs:24]: in the case we don't want music or sound effects to be cut off by each scene or restart

**Comment by GhostEntity12** [comment]
Superseded by https://github.com/GhostEntity12/HouseHunting/pull/28 - Audio Manager

**Pull request closed without merging**